src/main/java/com/github/nicholasmoser/gnt4/seq/ext/SeqEdit.java

```diff
@@ -110,7 +209,40 @@ public byte[] getFullBytes() {
       baos.write(ByteUtils.fromInt32(offset));
       baos.write(oldBytes);
       baos.write(STOP);
-      baos.write(newBytesWithBranchBack);
+      //if (newBytesWithBranchBack != null) {
+      //  baos.write(newBytesWithBranchBack);
+      //} else {
+        for (Opcode op : newCodes) {
+          baos.write(op.getBytes(position, size));
+        }
+        baos.write(branchBack);
+      //}
+      baos.write(STOP);
+      return baos.toByteArray();
+    } catch (Exception e) {
+      throw new RuntimeException(e);
+    }
+  }
+
+  /**
+   * @return The full bytes for this seq edit, to be appended to the seq extension section of the
+   * seq file.
+   */
+  public byte[] getFullBytes(int offset) {
```

Comment:
Can you add some javadoc about what this `offset` param represents? I'm trying to understand how this method differs from the one above and how `offset` differs from `this.offset`

Reply:
Changed naming of the parameter, should I add a longer explanation?